src/benchmark.cc

```diff
@@ -249,7 +262,23 @@ std::vector<BenchmarkReporter::Run> RunBenchmark(
       // clang-format on
 
       if (should_report) {
-        BenchmarkReporter::Run report = CreateRunReport(b, results, seconds);
+        MemoryManager::Result memory_result;
```

Comment:
@dominichamon why was this added here, and not at the end of the body of per-repetition loop?
```
     }
     // Here
   }
   // Calculate additional statistics	   // Calculate additional statistics
```